website/docs/release_notes.md

```diff
@@ -5,6 +5,178 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.13](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.13)
+
+## Change default command to `repl` when no args are passed
+
+The `run` command is now no longer the default when no arguments are passed, starting from `0.1.13` Scala CLI run scala `repl` under the hood.
+
+```
+$ scala-cli -S 3
+Welcome to Scala 3.1.3 (17.0.3, Java OpenJDK 64-Bit Server VM).
+Type in expressions for evaluation. Or try :help.
+
+scala> 
+```
+
+When inputs are provided, scala-cli uses the `run` command as default.
+
+```
+$ cat hello.sc
+println("Hello World")
+$ scala-cli hello.sc
+Hello World
+```
+
+This changed was added by [@Gedochao](https://github.com/Gedochao) in [#1268]( https://github.com/VirtusLab/scala-cli/pull/1268)
+
+## Optional project settings file
+
+Scala CLI now adds support for the optional configuration file `project.settings.scala` which determines the root workspace directory. The settings file is recommended place to put most using directives (dependencies, scala version, options).
```

Comment:
```suggestion
Scala CLI now supports marking the workspace root directory with an optional configuration file: `project.settings.scala`.  The workspace root determines where the `.bsp` and `.scala-build` directories will be saved (which mostly affects what path should be opened in your IDE to import the Scala CLI project through BSP).

The settings file is also the recommended input for your project's `using directives`. Otherwise, it functions similarly to other `.scala` sources.
```